app/ts/components/pages/WebsiteAccess.tsx

```diff
@@ -75,18 +73,20 @@ export function useWebsiteAccess() {
 
 export const WebsiteAccessView = () => {
 	return (
-		<WebsiteAccessProvider>
-			<Layout>
-				<Layout.Header>
-					<h1>Manage Websites</h1>
-					<SearchForm id = 'site_search' name = 'search' placeholder = 'Search website name, url or wallet address' />
-				</Layout.Header>
-				<Layout.Main>
-					<WebsiteSettingsList />
-					<WebsiteSettingsDetail />
-				</Layout.Main>
-			</Layout>
-		</WebsiteAccessProvider>
+		<AddressBookProvider>
+			<WebsiteAccessProvider>
+				<Layout>
+					<Layout.Header>
+						<h1>Manage Websites</h1>
+						<SearchForm id = 'site_search' name = 'search' placeholder = 'Search website name, url or wallet address' />
```

Comment:
```suggestion
						<SearchForm id = 'site_search' name = 'search' placeholder = 'Search website name, url or Ethereum address' />
```

Reply:
the PR was behind, will be fixed in upstream https://github.com/DarkFlorist/TheInterceptor/pull/1117